dataverse-webapp/src/main/webapp/resources/js/dv_rebind_bootstrap_ui.js

```diff
@@ -288,6 +288,28 @@ function reinitializePrimefacesComponentsJS() {
                             return;
                         }
 
+                        var totalSize = 0;
+                        const filesTable = PF('filesTable');
```

Comment:
not sure if we should assume `widgetVar` value. It's quite easy to miss this if we would want for example change the name of the widgetVar in some datatable

Reply:
I've changed to jq-selection by `data-currentBatchSize` attribute. It should be more reliable in that case.